corehq/apps/users/tests/test_models.py

```diff
@@ -13,6 +14,35 @@
 from corehq.apps.domain.models import Domain
 
 
+class HqPermissionsTest(SimpleTestCase):
+
+    def setUp(self):
+        self.hq_permissions = HqPermissions()
+        self.permission_names = self.hq_permissions.permission_names()
+        self.permissions = {name: getattr(self.hq_permissions, name) for name in self.permission_names}
+
+    def test_permissions_default_false(self):
+        default_true_permissions = ['access_all_locations', 'report_an_issue']
+        for name, value in self.permissions.items():
+            if name not in default_true_permissions:
+                self.assertFalse(value)
```

Comment:
You might as well add in `else: assertTrue` for those that do default to True

Reply:
resolved in [b366970](https://github.com/dimagi/commcare-hq/pull/32345/commits/b36697058214b20564c6519b56405d6d7d302bb5)